app.js

```diff
@@ -693,22 +697,67 @@ class App {
 
     this.scene.add(this.camera);
 
-    this.composer = new THREE.EffectComposer(this.renderer);
+    // Set up render target for holding depth information.
```

Comment:
It'd be great to make all the new composers/passes conditional on a url param, so if it's not enabled, the behavior/rendering is unchanged from before.  This will make it easier to deploy this new feature while we experiment with and tweak it, without changing the current user experience.  (At the moment it looks to me like there is a bit of a haze effect, or maybe it's just the antialising, I'm not sure, even if the new haze & depth params aren't present in the url.  Also, eventually we might want different haze settings for birds-eye vs street-level views.  In any case, if we can make it so that none of this is enabled, and the user sees the same result as before, unless they add special url params, then we can go ahead and deploy it and start experimenting with it.

Reply:
Good note, I'll work on that. I think the current "haze" when the haze feature is disabled actually comes from the softening of the edges with the antialising from a previous merge, but I'll double check that.

Reply:
I think the fuzziness is from the screen I captured it on, which has a lower resolution. I've taken another screenshot (attached) with all the new features disabled (which is the default). Certainly looks less fuzzy now I think.

![Screen Shot 2021-04-23 at 14 07 14](https://user-images.githubusercontent.com/9429048/115875742-6fc3df00-a43d-11eb-9b19-f8f8aedf9d0e.png)